### DISCUSSION TITLE
Type error on creating spacy document object

### DISCUSSION THREAD

**Discussion by DrZainulabdeen** [explosion/spaCy#11493] [Help: Other Questions] [answered]
I have scraped and saved a book as txt file with utf-8 encoding but loading this file as an spacy object gives type error

```
TypeError: '>' not supported between instances of 'int' and 'str'
```

I am not sure how a text file can have type error, you can download the book from [https://github.com/DrZainulabdeen/FireAndBlood-CharacterAnalysis/blob/master/data/fire-and-blood.txt](url)

```python
NER = spacy.load('en_core_web_sm')
NER.max_length = '2000000'

book_path = 'data/fire-and-blood.txt'

with open(book_path, 'r', encoding = 'utf-8') as f:
    book = f.read()
    book_doc = NER(book)
```

**Comment by rmitsch** [comment, marked as answer]
Hi @DrZainulabdeen, you assigned `NER.max_length` a string value. Try running it with `NER.max_length = 2000000`.
